Collateral Damage (CDMG): Add auto generation features

patschroed opened the pull request:
## Changes / Comments
- Added randomization feature that will allow players to more easily generate a character from the sheet.
- Updated sheet to work with CSE
- Updated styling to make intractable objects more clear

## Roll20 Requests

Comments are very helpful for reviewing the code changes. Please answer the relevant questions below in your comment.

- [x] Does the pull request title have the sheet name(s)? Include each sheet name.
- [ ] Is this a bug fix?
- [x] Does this add functional enhancements (new features or extending existing features) ?
- [x] Does this add or change functional aesthetics (such as layout or color scheme) ? 
- [ ] If changing or removing attributes, what steps have you taken, if any, to preserve player data ?
- [ ] If this is a new sheet, did you follow [Building Character Sheets standards](https://wiki.roll20.net/Building_Character_Sheets#Roll20_Character_Sheets_Repository) ?

If you do not know English. Please leave a comment in your native language.

roll20deploy commented:
[Character Sheet Info](https://app.roll20.net/managesheets/CDMG/roll20-character-sheets) *Roll20 Internal Use only.*

patschroed commented:
Does this add functional enhancements (new features or extending existing features)? I've updated the sheet to include content that will allow us to randomize the fields based on content from the source material.

Does this add or change functional aesthetics (such as layout or color scheme)? I've updated the styling to be clearer for intractable objects and added new buttons for the new features.
